test.sh

```diff
@@ -139,7 +140,7 @@ cd ${go_workspace}/src/sigs.k8s.io/kubebuilder
 go test ./cmd/... ./pkg/...
 
 # test project v1
-test_project project 1
+GO111MODULE=auto test_project gopath/src/project 1
```

Comment:
Comment explaining use of `auto` for `GO111MODULE` will help why is that being done for version 1 project.